scripts/mintmaker.sh

```diff
@@ -204,6 +204,21 @@ for pr in $PR_NUMS; do
     # attempt cherry-pick
     if git cherry-pick "$sha"; then
       echo "    cherry-pick succeeded."
+      CHANGES=$(git show --pretty=format:"" --name-only "${sha}" -- go.{mod,sum} */go.{mod,sum})
+      if [[ -n "${CHANGES}" ]]; then
+        # run go mod tidy where applicable
+        for gomod in $(echo ${CHANGES} | xargs -n1 dirname | uniq); do
+          (cd "${gomod}" && go mod tidy)
+        done
+        if [[ -n "$(git diff --ignore-submodules)" ]]; then
+          echo "   go modules are not tidy :"
+          git diff --ignore-submodules
+          echo "   skipping."
+	  git reset --hard --quiet HEAD^
```

Comment:
nit: indentation

Reply:
Dammit... emacs switched back to tabs despite I asked for spaces only !